app/actions/pendingTransactionActions.js

```diff
@@ -136,15 +136,72 @@ export const pruneConfirmedOrStaleTransaction = async (
   if (Array.isArray(storage[address])) {
     storage[address] = storage[address].filter(
       // use includes here to be indifferent to 0x prefix
-      transaction => !transaction.hash.includes(txId),
+      transaction => transaction.hash && !transaction.hash.includes(txId),
     )
   }
   await setPendingTransactions(storage)
 }
 
+export const fetchTransactionInfo = async (
+  transactions: PendingTransactions,
+  address: string,
+  net: string,
+) => {
+  if (Array.isArray(transactions[address]) && transactions[address].length) {
```

Comment:
is it ever possible that pending transactions is undefined instead of an empty object? this would cause a problem

Reply:
👍 by default electron JSON storage will always return an empty object for the key value pairing but I will go ahead and add a check in here anyway